pilot/docker/envoy_pilot.json

```diff
@@ -21,33 +21,6 @@
     ],
     "bind_to_port": true
    },
-   {
-    "address": "tcp://0.0.0.0:15005",
```

Comment:
why that change?

Reply:
Costin, this broke the feature we talked about, i.e., use 15005 for https connection and 15007 for http connection.